### PR TITLE
Initial docs

### PR DESCRIPTION
**Pull request by devin-petersohn**
Creating the initial documentation for Readthedocs.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Installation
+============
+
+There are a couple of ways to start using Pandas on Ray. Most users will want to
+install with ``pip``, but some users may want to build from the master branch on
+the `GitHub repo`_.
+
+Installing with pip
```
Comment (kunalgosar) [inline docs/installation.rst]: Do we want to add documentation on the ray requirement here?

File: docs/pandas_on_ray.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+In order to use the most up-to-date version of Pandas on Ray, please follow
+the instructions on the `installation page`_
+
+Once you import the library, you should see something similar to the following
+output:
+
+.. code-block:: text
+
+  >>> import ray.dataframe as pd
```
Comment (kunalgosar) [inline docs/pandas_on_ray.rst]: This is outdated.

File: docs/pandas_on_ray.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+distribute the data. In fact, users can continue using their previous Pandas
+notebooks while experiencing a considerable speedup from Pandas on Ray, even
+on a single machine. Only a modification of the import statement is needed, as
+we demonstrate below. Once you’ve changed your import statement, you’re ready
+to use Pandas on Ray just like you would Pandas.
+
+.. code-block:: python
+
+  # import pandas as pd
+  import ray.dataframe as pd
```
Comment (kunalgosar) [inline docs/pandas_on_ray.rst]: This is outdated.

File: docs/pandas_on_ray.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  Waiting for redis server at 127.0.0.1:14618 to respond...
+  Waiting for redis server at 127.0.0.1:31410 to respond...
+  Starting local scheduler with the following resources: {'CPU': 4, 'GPU': 0}.
+
+  ======================================================================
+  View the web UI at http://localhost:8889/notebooks/ray_ui36796.ipynb?token=ac25867d62c4ae87941bc5a0ecd5f517dbf80bd8e9b04218
+  ======================================================================
+
+Once you have executed  ``import ray.dataframe as pd``, you're ready to begin
```
Comment (kunalgosar) [inline docs/pandas_on_ray.rst]: Same as above.

File: docs/pandas_on_ray.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+Examples
+--------
+You can find an example on our recent `blog post`_ or on the
+`Jupyter Notebook`_ that we used to create the blog post.
+
+.. _`installation page`: http://ray.readthedocs.io/en/latest/installation.html
+.. _`here`: http://ray.readthedocs.io/en/latest/pandas_supported.html
+.. _`open an issue`: http://github.com/ray-project/ray/issues
+.. _`blog post`: http://rise.cs.berkeley.edu/blog/pandas-on-ray
```
Comment (kunalgosar) [inline docs/pandas_on_ray.rst]: Does the code in the blog post need to be refactored to the new modin namespace?

File: docs/pandas_supported.rst [hunk 6]
```diff
@@ -0,0 +1,756 @@
...
+methods and APIs are merged into the master branch, and not necessarily correct
+as of the most recent release. In order to install the latest wheels, follow
+the directions found here_
+
+Questions on implementation details
+-----------------------------------
+
+If you have a question about the implementation details or would like more
+information about an API or method in **Pandas on Ray**, please contact the ray
+developer `mailing list`_.
```
Comment (kunalgosar) [inline docs/pandas_supported.rst]: This needs to be modin mailing list

File: docs/pandas_supported.rst [hunk 7]
```diff
@@ -0,0 +1,756 @@
...
+| ``to_stata``              | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_string``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_timestamp``          | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_xarray``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``transform``             | Y                  |                                                    |
++----
```
Comment (kunalgosar) [inline docs/pandas_supported.rst]: This is outdated

Comment (kunalgosar) [inline docs/pandas_supported.rst]: The links here need to be updated.

File: docs/pandas_on_ray.rst (hunk 5 above)
Comment (devin-petersohn) [reply docs/pandas_on_ray.rst]: It will be, yes.

File: docs/installation.rst (hunk 1 above)
Comment (devin-petersohn) [reply docs/installation.rst]: Added, thanks!

File: docs/pandas_supported.rst [hunk 8]
```diff
@@ -0,0 +1,756 @@
...
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_stata``              | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_string``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_timestamp``          | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``to_xarray``             | N                  | N/A, Not Yet Implemented                           |
++---------------------------+--------------------+----------------------------------------------------+
+| ``transform``             | Y                  |
```
Comment (kunalgosar) [inline docs/pandas_supported.rst]: This link needs to be updated.

Comment (kunalgosar) [inline docs/pandas_supported.rst]: Same as above

**Pull request merged**